README.md

```diff
@@ -164,61 +170,69 @@ You can verify the installation by running:
 
 ## How to develop
```

Comment:
Just a personal nit that I usually put this in a separate `DEVELOP.md` file because it's not directly relevant to end-users, but it's ok to leave it here too